.devcontainer/scripts/minimal_requirements.txt

```diff
@@ -0,0 +1,5 @@
+dbt-core==1.8.2
+dbt-duckdb==1.8.1
+pre-commit==3.7.1
+black
+python-dotenv
```

Comment:
I am unsure of the purpose of the root `requirements.txt` too? but I think thats for a different PR!

Reply:
A devcontainer is for a consistent development experience - and `black` and `pre-commit` are the bare minimum and ensure that code contributions meet a minimum standard. We should have `sqlfluff` included explicitly in there too - I think it gets installed anyway as part of dbt.

I can see that the `pre-commit -config.yaml` file is very minimal - and can be improved as well. Possibly a subset of what is here -> https://github.com/vvcb/dbt-synthea/blob/vc/databricks/.pre-commit-config.yaml

Typically, one wouldn't need a separate requirements file within the devcontainer and the project `requirements.txt` should be sufficient. The new PR could address this with a single requirements.txt file for the project with database-specific dependencies and dependencies required only for development (such as black, pre-commit, etc.) specified as optional dependencies. This should allow the user to install the project with `pip install dbt_synthea[postgres,dev]` for instance. 

Development dependencies are going to be different from dependencies required for just running the code.

Reply:
Thank you for your explanation.

Ah that makes more sense, I trimmed out some of it as it wasn't referenced anywhere else and didn't see why it would be needed ~ it's probably fine to leave it in for now and can have another PR to tighten up precommit actions

Reply:
can/should this be updated now that #67 is merged in?